ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.10.yaml

```diff
@@ -51,11 +51,19 @@ tests:
   steps:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-proxy
-- as: e2e-openstack-nfv
+- as: e2e-openstack-nfv-intel
   interval: 72h
   steps:
     cluster_profile: openstack-nfv
     workflow: openshift-e2e-openstack-nfv
```

Comment:
Shouldn't the workflow here be` openshift-e2e-openstack-dpdk` to allow testing dpdk with intel? and including the appropriate Networks.